src/status_im/contexts/wallet/send/select_collectible_amount/view.cljs

```diff
@@ -11,24 +12,20 @@
 (defn view
   []
   (let [on-close              (rn/use-callback #(rf/dispatch [:navigate-back]))
-        [value set-value]     (rn/use-state 1)
-        inc-value             (rn/use-callback (fn [] (set-value (inc value)))
-                                               [value])
-        dec-value             (rn/use-callback (fn [] (set-value (dec value)))
-                                               [value])
-        add-digit             (rn/use-callback (fn [digit]
-                                                 (set-value (+ (js/parseInt digit)
-                                                               (* value 10))))
-                                               [value])
-        delete-digit          (rn/use-callback (fn []
-                                                 (set-value (Math/floor (/ value 10))))
-                                               [value])
-
+        [value set-value]     (rn/use-state controlled-input/init-state)
         send-transaction-data (rf/sub [:wallet/wallet-send])
         collectible           (:collectible send-transaction-data)
         balance               (utils/collectible-balance collectible)
         preview-uri           (get-in collectible [:preview-url :uri])
-        incorrect-value?      (or (< value 1) (> value balance))]
+        incorrect-value?      (controlled-input/input-error value)]
+    (rn/use-mount
+     (fn []
+       (set-value #(controlled-input/set-numeric-value % 1))
+       (set-value #(controlled-input/set-lower-limit % 1))))
```

Comment:
Why do we `set-value` at unmount?

Could you please explain it @vkjr  :)

Reply:
It's a `mount`, but still why? 🤔

Reply:
It's a `mount`, but still why? 🤔

Reply:
It's a mount, but still why? 🤔